### DISCUSSION TITLE
Blog post about Seabird

### DISCUSSION THREAD

**Discussion by seifrajhi** [getseabird/seabird#30] [General]
Hello,

I really like this tool, thank you for the efforts, after being let down by Lens.. I hope as promised it stays forever open source,
So, now I plan to write blog post about, is that ok ?

My blog post: https://medium.com/@seifeddinerajhi

**Comment by jgillich** [comment]
Hi, of course it's ok :) Let me know if you have any questions

**Reply by seifrajhi** [reply]
Thank you @jgillich , I am sure after sharing I will get many questions, so let me get ready for that,
Are you planning to extend the docs/README ?
Also, do you have timeline like EDA to fix the authentication issue with EKS and AKS ?

**Reply by jgillich** [reply]
I'll probably add a documentation section to the website in the future, but I'd consider it a UX failure if the software needed a manual.

> authentication issue with EKS and AKS

Exec auth plugin support? That will be included in the next release, likely going out by the end of the week.
